docs/guide/docs/releases/migrate.md

```diff
@@ -1,37 +1,68 @@
 ---
 id: migrate
-title: Migrations
+title: Migration & Updates
 ---
 
-## Migration from 11.9 to 12.0
+## Changelog and Release Notes
```

Comment:
Can you also point to individual releases [here](https://github.com/botpress/botpress/releases)